### PR TITLE
Add TagBuilder copy constructor (#10955)

### PR DESCRIPTION
**Pull request by fstaffa**
Added copy constructor to TagBuilder (#10955 )

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/aspnetcore?pullRequest=22729) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/aspnetcore?pullRequest=22729) before we can accept your contribution.<br/><br/>:x: fstaffa [sign now](https://cla.dotnetfoundation.org/dotnet/aspnetcore?pullRequest=22729)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/aspnetcore?pullRequest=22729) it.</sub>

File: src/Mvc/Mvc.ViewFeatures/src/Rendering/TagBuilder.cs [hunk 1]
```diff
@@ -37,6 +37,30 @@ public TagBuilder(string tagName)
             TagName = tagName;
         }
 
+        public TagBuilder(TagBuilder tagBuilder)
```
Comment (pranavkm) [inline src/Mvc/Mvc.ViewFeatures/src/Rendering/TagBuilder.cs:40]: Can you document this ctor?

Comment (fstaffa) [reply src/Mvc/Mvc.ViewFeatures/src/Rendering/TagBuilder.cs:44]: Added documentation

**Review by pranavkm — APPROVED** [review]
Looks great. Thanks for the PR!

**Comment by NickCraver** [comment]
I'll ty and play with it this week - honestly don't have the original code from when we discussed and we had moved off due to the lack of it. I'll revisit!

**Comment by pranavkm** [comment]
Still open to feedback @NickCraver. I'll merge this in to avoid the PR going stale.

**Pull request merged**